src/database/migrations/0093_auto_20210607_0945.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.0.14 on 2021-06-07 16:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('database', '0092_auto_20210604_1348'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='userprofile',
+            name='auto_icon',
+        ),
+        migrations.AddField(
+            model_name='userprofile',
+            name='color',
+            field=models.CharField(default='#000000', max_length=7),
+        ),
+    ]
```

Comment:
I would consolidate the three migration files, by deleting them and regenerating